locales/fr/translation.json

```
@@ -365,11 +365,11 @@
   "Looks like you've already sent an invitation to that email.": "Il semblerait que vous avez déjà envoyé une invitation sur cet email",
   "Looks like your e-mail address has not been verified.": "Il semblerait que votre adresse email n'a pas été vérifiée",
   "Loop mode": "",
-  "Loop mode off": "Loop mode désactivé",
+  "Loop mode off": "Loop mode arrêté",
   "low": "bas",
   "Low": "",
   "Lower target must be less than upper target.": "La limite basse doit être inférieure à la limite haute",
-  "M_Label": "D",
+  "M_Label": "S",
```

From the Jira issue:

> as well as the letter "D" > "A"
```suggestion
  "M_Label": "A",
```